src/main/java/dev/aurelium/auramobs/AuraMobs.java

```diff
@@ -79,6 +80,9 @@ public void onEnable() {
         language = new Locale(optionString("language"));
         mobKey = new NamespacedKey(this, "isAureliumMob");
         namesEnabled = optionBoolean("custom_name.enabled");
+
+        ScaleUtil.initialize(this);
```

Comment:
To keep the project consistent with my code standard/style, instead of making static calls, ScaleUtil should be renamed to ScaleManager with non-static fields and methods. Then, instantiate an instance of ScaleManager in onEnable and store it as a field in AuraMobs, with accesses outside the class using a new getScaleManager method. Also, move the ScaleManager class to the entities package.